### PR TITLE
Fix height for comment anchors in request history

### PR DESCRIPTION
**Pull request by rubhanazeem**

### CODE REVIEW

**Comment by rubhanazeem** [comment]
https://trello.com/c/m3aezM5E/2681-comment-anchors-dont-account-for-the-height-of-the-top-navbar

**Comment by rubhanazeem** [comment]
[Comment anchors don't account for the height of the top navbar](https://trello.com/c/m3aezM5E/2681-comment-anchors-dont-account-for-the-height-of-the-top-navbar)

File: src/api/app/assets/javascripts/webui/request.js [hunk 1]
```diff
@@ -138,6 +138,10 @@ $(document).ready(function(){
   $('#changes-item').on('shown.bs.tab', function () {
     loadChanges();
   });
+
+  var shiftWindow = function() { scrollBy(0, -57) };
```
Comment (dmarcoux) [inline src/api/app/assets/javascripts/webui/request.js:142]: We definitely a comment here for this, especially for the `-57`. What is this value exactly? The height of the navbar?

Comment (rubhanazeem) [reply src/api/app/assets/javascripts/webui/request.js:142]: Yes, the height is 57

Comment (hennevogel) [reply src/api/app/assets/javascripts/webui/request.js:142]: Hm, isn't this broken for every anchor on every page?

Comment (dmarcoux) [reply src/api/app/assets/javascripts/webui/request.js:142]: Yes and with this approach, it solves the issue, but only partially. For example, if you click on an anchor, then scroll a bit to look at something else, then click on the anchor again, it's broken again:

preview

Comment (rubhanazeem) [reply src/api/app/assets/javascripts/webui/request.js:142]: I'm not sure if this is broken for every anchor. Do you have any examples where it is broken outside this page?

Comment (dmarcoux) [reply src/api/app/assets/javascripts/webui/request.js:142]: Yes, in the comments on the project#show page. Same for the package#show page.

**Comment by dmarcoux** [comment]
As we discussed, we're closing this. We'll look for another solution.

**Pull request closed without merging**